index.js

```diff
@@ -2,16 +2,35 @@
 const path = require('path');
 const eslint = require('eslint');
 const globby = require('globby');
+const multimatch = require('multimatch');
 const optionsManager = require('./options-manager');
 
 exports.lintText = (str, opts) => {
 	opts = optionsManager.preprocess(opts);
 
+	if (opts.cwd && opts.filename) {
+		const filename = path.relative(opts.cwd, opts.filename);
+
+		if (multimatch([filename], opts.ignores)) {
+			return {
+				errorCount: 0,
+				warningCount: 0,
+				results: [{
+					errorCount: 0,
+					filePath: filename,
+					messages: [],
+					warningCount: 0
+				}]
+			};
+		}
+	}
```

Comment:
This needs to take into account the overrides logic and should be after the below overrides logic. Right now it wouldn't work if you used config overrides that enabled it again. Should also have a test for that to ensure it works.